Dockerfile

```diff
@@ -52,6 +52,9 @@ COPY --from=build-stage0 /aws/bin/  /usr/local/bin
 COPY --from=build-stage0 /usr/local/aws-cli /usr/local/aws-cli
 COPY scripts /managed-scripts
 
+# Install python packages
+RUN python3 -m pip install -r /managed-scripts/CSSRE/requirements.txt --user
```

Comment:
seems to be awfully specific file ..
should these packages be hardcoded into the Dockerfile so all will be able to use them?

this is done so changes in  `/managed-scripts/CSSRE/requirements.txt` won't cause voodoo and add stuff to our container without us recognizing it

Reply:
@georgettica 
> should these packages be hardcoded into the Dockerfile so all will be able to use them?
This was not the intention. And the upside of using the file is that it is easy to tell which team provided these packages, secondly, we don't have to build a container to run a simple python script in our local machine. IMO putting it at the top-level doesn't guarantee from someone adding "voodoo", everything goes to a PR anyway. 

However if putting every package in the dockerfile is the pattern we want to follow, then we can update this.

Reply:
@georgettica modified the `Dockerfile` with required packages, rather than using `requirements.txt` file

Reply:
+1 for adding requests and tabulate python packages. 
My preference via `dnf install python3-tabulate.noarch python3-requests` but I guess `pip3 install requests tabulate` would work too.